### ISSUE TITLE
[Accessibility]: Exiting web area from ref panel close button doesn't allow re-entry

### ISSUE THREAD

**Issue by varungandhi-src** [sourcegraph/sourcegraph#36571]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33510

### Problem description

When the focus is on the ref panel close button (X), the screen reader announces a keyboard shortcut to exit the web area (Control + Option + Shift + ↑). However, if you type that shortcut, then trying Control + Option + Shift + ↓ doesn't bring you back to the ref panel close button, it takes you to a different part of the page.

### Expected behavior

I don't know if this is expected behavior for screen reader users, but it seems a little counter-intuitive from my POV; I would've expected that re-entering should "just work."

### Additional details

You can trigger the ref panel by clicking on Find References after hovering over an identifier. ([example link](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/cmd/server/internal/goreman/log.go?L23:5&subtree=true#tab=references)). The Close button is near the right side.

![](https://user-images.githubusercontent.com/93103176/171936416-3cac8a5a-7f40-4eb7-90ff-c3f7d2662c35.png)

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

I'm unsure who should work on this issue or if it needs more clarification. I will let the Frontend Platform team triage this.

**Comment by umpox** [comment]
I think this is expected behavior for screen readers (AFAIK this shortcut usually triggers changing between windows, e.g. browser -> webpage -> iframe page).

Unsure why this is happening for this button specifically, will investigate further

**Comment by umpox** [comment]
Closing, after more investigation this is a general browser shortcut rather than in-page shortcut. I don't think users would expect this behavior
